platform/server/src/test/java/org/motechproject/server/it/MessageBrokerPingServiceIT.java

```diff
@@ -0,0 +1,53 @@
+package org.motechproject.server.it;
+
+import junit.framework.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.motechproject.server.bootstrap.MessageBrokerPingService;
+import org.motechproject.server.bootstrap.MessageBrokerPingServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+/**
+ * Created by atish on 18/7/15.
+ */
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration
+public class MessageBrokerPingServiceIT {
```

Comment:
should probably just leave this test since its almost the same as above, just another reason to merge the classes